.github/workflows/release.yml

```diff
@@ -0,0 +1,74 @@
+name: Build and publish Python distribution to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  build:
+    name: Build distribution
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v5.1.0
+      with:
+        python-version: "3.x"
+
+    - name: Install pypa/build
+      run: >-
+        python3 -m
+        pip install
+        build
+        --user
```

Comment:
Why is this line broken like this? :thinking:

Reply:
I don't know, that's how it is in the guide. I guess we can change it.

Reply:
Yeah, if we can't figure out a good reason for that, I'd say we should change it. While you're at it, please change "Github" to "GitHub" in the commit message :)